packages/concerto-tools/lib/codegen/fromcto/jsonschema/jsonschemavisitor.js

```diff
@@ -78,28 +80,36 @@ class JSONSchemaVisitor {
      * @public
      */
     visit(thing, parameters) {
-        if (thing.isModelManager?.()) {
-            return this.visitModelManager(thing, parameters);
-        } else if (thing.isModelFile?.()) {
-            return this.visitModelFile(thing, parameters);
-        } else if (thing.isAsset?.()) {
-            return this.visitAssetDeclaration(thing, parameters);
-        } else if (thing.isTransaction?.()) {
-            return this.visitTransactionDeclaration(thing, parameters);
-        } else if (thing.isEnum?.()) {
-            return this.visitEnumDeclaration(thing, parameters);
-        } else if (thing.isConcept?.()) {
-            return this.visitConceptDeclaration(thing, parameters);
-        } else if (thing.isClassDeclaration?.()) {
-            return this.visitClassDeclaration(thing, parameters);
-        } else if (thing.isField?.()) {
-            return this.visitField(thing, parameters);
-        } else if (thing.isRelationship?.()) {
-            return this.visitRelationshipDeclaration(thing, parameters);
-        } else if (thing.isEnumValue?.()) {
-            return this.visitEnumValueDeclaration(thing, parameters);
-        } else {
-            throw new Error('Unrecognised type: ' + typeof thing + ', value: ' + util.inspect(thing, { showHidden: true, depth: null }));
+        try{
```

Comment:
Why do you need to catch and swallow the error, please?

This is would be a breaking change.